datadog_cluster_agent/datadog_checks/datadog_cluster_agent/data/auto_conf.yaml

```diff
@@ -0,0 +1,20 @@
+## @param ad_identifiers - list of strings - required
+## A list of container identifiers that are used by Autodiscovery to identify
+## which container the check should be run against. For more information, see:
+## https://docs.datadoghq.com/agent/guide/ad_identifiers/
+#
+ad_identifiers:
+  - cluster-agent
+
+## All options defined here are available to all instances.
+#
+init_config:
+
+## Every instance is scheduled independent of the others.
+#
+instances:
+
+    ## @param prometheus_url - string - required
+    ## The URL where your application metrics are exposed by Prometheus.
+    #
+  - prometheus_url: http://%%host%%:5000/metrics
```

Comment:
Let's also add the `is_leader` label joins config to this config, wdyt?

Reply:
Confirmed that the default config will still apply

Reply:
Yes, I also added the changes in the spec.yaml so the ad template has the same options as the default one.